src/globals.d.ts

```diff
@@ -1,3 +1,5 @@
+/* tslint:disable */
```

Comment:
TSLint was complaining that there are multiple file definitions in this file, and I would say it's OK for now.